app/models/post.rb

```diff
@@ -1,6 +1,12 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
 
+  # validations
+
+  validates :Title, presence: true, length: { maximum: 250 }
```

Comment:
Optional 
- [ ] I can see you create a column with a capital letter. To best practice, you can use a small letter.
Kindly check this [article](https://www.sisense.com/blog/better-sql-schema/).